indexer.ts

```diff
@@ -0,0 +1,743 @@
+              await handleNewProject(
+                args,
+                action.receiverId,
+                action.signerId,
+                action.receiptId
+              );
+              break;
+            
+            // chefs approve/decline ... etc a projects application to a pot
+            case "chef_set_application_status":
+              console.log(
+                "application status change incoming:",
+                JSON.parse(args)
+              );
+              await handleApplicationStatusChange(
+                args,
+                action.receiverId,
+                action.signerId,
+                action.receiptId
+              );
+              break;
+            
+            // registries can have default status for projects
+            case "admin_set_default_project_status":
+              console.log(
+                "registry default status setting incoming:",
+                JSON.parse(args)
+              );
+              await handleDefaultListStatusChange(
+                args,
+                action.receiverId,
+                action.signerId,
+                action.receiptId
+              );
+              break;
+            
+            // admins can set a project's status
+            case "admin_set_project_status":
+              console.log(
+                "project registration status update incoming:",
+                JSON.parse(args)
+              );
+              await handleProjectRegistrationUpdate(
+                args,
+                action.receiverId,
+                action.signerId,
+                action.receiptId
+              );
+              break;
+            
+            // fires when chef set payouts
+            case "chef_set_payouts":
+              console.log(
+                "setting payot....:",
+                JSON.parse(args)
+              );
+              await handleSettingPayout(
+                args,
+                action.receiverId,
+                action.signerId,
+                action.receiptId
+              );
+              break;
+            
+            // fires when there is a payout challenge
+            case "challenge_payouts":
+              console.log(
+                "challenge payout:",
+                JSON.parse(args)
+              );
+              await handlePayoutChallenge(
+                args,
+                action.receiverId,
+                action.signerId,
+                action.receiptId
+              );
+              break;
+            // fires when fulfilling payouts
+            case "transfer_payout_callback":
+              console.log(
+                "fulfilling payouts.....",
+                JSON.parse(args)
+              );
+              await handlePayout(
+                args,
+                action.receiverId,
+                action.signerId,
+                action.receiptId
+              );
+              break;
+            case "owner_remove_admins":
```

Comment:
FYI there is a method with this name on all the contracts, so will need to be more specific to handle this based on the contract (registry/lists, pot factory, pot, donation)